spec/integration/bundler_spec.rb

```diff
@@ -0,0 +1,19 @@
+RSpec.describe 'Bundler' do
+  before :all do
+    @ruby = RbConfig.ruby.shellescape
+    @pry_dir = File.expand_path(File.join(__FILE__, '../../../lib')).shellescape
+  end
+
+  specify 'a LoadError is not raised after bundler has been activated' do
```

Comment:
The test looks good but the description is obscure. How about:

```
context "when Pry requires Gemfile, which doesn't specify Pry as a dependency"
  it "loads auto-completion correctly"
```

(or something like that)